app/src/main/java/io/homeassistant/companion/android/qs/TileExtensions.kt

```diff
@@ -112,7 +112,7 @@ abstract class TileExtensions : TileService() {
         ) {
             Log.d(TAG, "Click detected for tile ID: $tileId")
             val tileDao = AppDatabase.getInstance(context).tileDao()
-            val tileData = tileDao.get(tileId)
+            val tileData = tileDao.getSync(tileId)

```

Comment:
This whole function should become a suspend function but since all you are actually doing is a rename it's fine.